angular-rails-app/app/controllers/owners_controller.rb

```diff
@@ -3,10 +3,15 @@ class OwnersController < ApplicationController
    before_action :set_owner, only: [:show]
 
   def index
-    @owners = Owner.all
+    if params[:name]
+       @owners = Owner.where("name LIKE (?)", "%#{params[:name]}%").limit(6)
```

Comment:
@chrishenry  Actually, I'm not sure why this didn't occur to me before, but perhaps a more simple answer for now is just to concat `name` + `corporation_name` and display both in the dropdown box.  I think I'm going to try this...